storage/file.go

```diff
@@ -114,6 +132,24 @@ func (f *File) Create(maxSize uint64) error {
 	return nil
 }
 
+// CopyFile operation copied a file/blob from the sourceURL to the path provided.
+//
+// See https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/copy-file
+func (f File) CopyFile(sourceURL string, options *FileRequestOptions, metadata map[string]string) error {
```

Comment:
Agreed. This was a bad mistake by me. The splitting of file.go out to directory/share/file and services jumbled up a bunch of stuff. Fixing those :)